src/main/java/io/anserini/qa/passage/WmdPassageScorer.java

```diff
@@ -0,0 +1,7 @@
+package io.anserini.qa.passage;
```

Comment:
Why not use a more standard list, e.g., one from NLTK?
http://www.nltk.org/book/ch02.html